modules/ossm-enabling-sidecar-injection.adoc

```diff
@@ -0,0 +1,141 @@
+// Module included in the following assemblies:
+// install/ossm-sidecar-injection
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-enabling-sidecar-injection_{context}"]
+= Enabling sidecar injection
+
+The sidecar injection process in {istio} uses label-based rules to determine if a pod should be injected with a sidecar. If a namespace has both the `istio-injection` label and the `istio.io/rev` label, the `istio-injection` label takes precedence, treating the namespace as part of the default revision.
```

Comment:
This paragraph explains that labels are used for injection, but then instead of explaining which labels should be used, it explains what happens if both labels are used. This jump feels odd. Maybe rephrase the paragraph so that the two labels are mentioned first and then explain what happens if both are used (but this is actually not even that important).

Reply:
@luksa Rephrased it. Kindly check.